libs/methodologies/bold/rule-processors/mass-id/avoided-emissions/src/avoided-emissions.processor.ts

```diff
@@ -32,29 +34,37 @@ import {
   type RuleOutput,
   RuleOutputStatus,
 } from '@carrot-fndn/shared/rule/types';
+import { is } from 'typia';
 
+import { AVOIDED_EMISSIONS_BASELINE_MATRIX } from './avoided-emissions.constants';
 import { AvoidedEmissionsProcessorErrors } from './avoided-emissions.errors';
+import {
+  type RuleSubject,
+  WasteGeneratorBaselinesValue,
+} from './avoided-emissions.types';
 
-const { EMISSION_FACTOR } = DocumentEventAttributeName;
+const { BASELINES, EXCEEDING_EMISSION_COEFFICIENT } =
+  DocumentEventAttributeName;
+const { RECYCLING_BASELINE } = DocumentEventName;
 
 export const RESULT_COMMENTS = {
   APPROVED: (
     avoidedEmissions: number,
-    emissionIndex: number,
+    exceedingemissionCoefficient: number,
     currentValue: number,
   ) =>
-    `The avoided emissions were calculated as ${avoidedEmissions} kg CO₂e using the formula ${emissionIndex} × ${currentValue} = ${avoidedEmissions} [formula: emission_index × current_value = avoided_emissions].`,
-  MISSING_EMISSION_FACTOR: `The "${EMISSION_FACTOR}" attribute was not found in the "Recycler Homologation" document or it is invalid.`,
+    `The avoided emissions were calculated as ${avoidedEmissions} kg CO₂e using the formula (1 - ${exceedingemissionCoefficient}) × ${currentValue} = ${avoidedEmissions} [formula: (1 - emission_index) × current_value = avoided_emissions].`,
```

Comment:
Is this formula actually correct? I feel like it is missing the _avoidedEmissionsByMaterialAndBaselinePerTon_.